OpenRA.Mods.Common/Util.cs

```diff
@@ -175,5 +175,15 @@ public static int RandomDelay(World world, int[] range)
 
 			return world.SharedRandom.Next(range[0], range[1]);
 		}
+
+		public static WDist LargestValidHealthRadius(Ruleset rules, ActorInfo ai, bool onlyBlockers = false)
```

Comment:
I think this name is potentially misleading.  The `OuterRadius` != the health radius, consider a shape that is long and skinny.  A projectile travelling in one direction through the radius might hit it, but at the same distance from another direction might not.  I suggest something like `MinimumScanRadius`.